src/pages/TermsOfServiceAcceptance.jsx

```diff
@@ -19,6 +20,9 @@ export default function TermsOfServiceAcceptance(props) {
 
   const acceptToS = useCallback(async () => {
     await TosService.acceptTos();
+    // TODO: Determine if this is the correct user flow
+    // Can the user only accept ToS once the have signed in?
+    // What state is the user in after signIn, but before accepting ToS?
```

Comment:
This is the correct flow. Yes, the user can only accept ToS after signing in - otherwise we don't know who they are and could not register their ToS acceptance in Sam. If they sign in, but have not yet accepted ToS, they are directed to this page where their only choice is to accept or reject.

Reply:
That makes sense. I was trying to determine if the ToS accept button should set the `setUserIsLogged` to true, since by signing in, userIsLogged should already be set to true. With this context, I believe the ToS accept button should not change the userIsLogged state.

Reply:
Correct - `userIsLogged` state is separate and distinct from ToS acceptance.